src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/index.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import _ from 'lodash';
```

Comment:
I don't see this used? o.o

Reply:
@billyvg fixed. I guess the linter ignored it since it starts with "_".